services/editor/src/components/ErrorPage.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+// import styled from 'react-emotion';
+import styled from '@emotion/styled';
+
+import logoSrc from './resources/logo.svg';
+
+const getErrorText = (error) => {
+  if (error.statusText) {
+    return error.statusText;
+  } else if (error.response && error.response.statusText) {
+    return error.response.statusText;
+  }
+  return 'Unknown Error';
+};
+
+const Main = styled.div`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+`;
+
+const Header = styled.div`
+  top: 0px;
+  left: 0px;
+  right: 0px;
+  height: 70px;
+  background-color: #009fda;
+
+  display: flex;
+  align-items: center;
+`;
+
+const Logo = styled.img`
+  margin: 5px 0 0 20px;
+  width: 140px;
+`;
+
+const Body = styled.div`
+  position: absolute;
+  top: 70px;
+  left: 0px;
+  right: 0px;
+  bottom: 0px;
```

Comment:
maybe you can just use `flex: 1;` instead of all these props.
an issue you might have is when Body content is too long the scrolling won't work properly.

Reply:
If I set it to flex: 1, the height becomes height of the inner component. After a quick check, I saw that people suggest this method for similar situations (setting position absolute to make child height 100% and the play with top/left/etc for central alignment).